web/docs/guides/local-development.mdx

```diff
@@ -124,6 +124,18 @@ http://localhost:54321/auth/v1/           # Auth (GoTrue)
 </Tabs>
 
 
+### Available services
+
+
+When working with local environment, not every service is available for now. Here are the ones that are *NOT* availables:
```

Comment:
Hi @loicknuchel - these services ARE availble, it's just that they are not on the Studio interface (for now)

For example, you can set up Social Auth using the GoTrue config/settings. Same with Storage - it is available through the API.

I will leave this open, because I think you're correct about making this clear from the outset

Reply:
Thanks for answering.

As they are not present in the studio, how can they be configured?
Do you have a documentation about this?

Reply:
Using Environment variables: https://github.com/supabase/supabase/blob/master/docker/.env.example

https://supabase.com/docs/guides/hosting/overview#configuration